Framework/MDAlgorithms/inc/MantidMDAlgorithms/IntegratePeaksMD2.h

```diff
@@ -79,7 +79,7 @@ class DLLExport IntegratePeaksMD2 : public API::Algorithm {
 
   /// Calculate if this Q is on a detector
   void calculateE1(const Geometry::DetectorInfo &detectorInfo);
-  double detectorQ(Mantid::Kernel::V3D QLabFrame, double r);
+  double calculateDistanceToEdge(Mantid::Kernel::V3D QLabFrame, double r);
```

Comment:
This predates your changes but the QLabFrame parameter should be const reference I think. The function itself should also be labelled const (at the end)

Reply:
Thanks, yes you are correct and I have also removed the input `r` as I can just initialise with `edgeDist` with `DBL_MAX`